project/ndaparser/parser.py

```diff
@@ -126,6 +127,22 @@ def ascii2scandic(string):
     return string
 
 
+def parseCsv(f):
+    transactions = []
+    for t in DictReader(f, delimiter=';'):
+        if not t["Viitenumero"]:
+            continue
+        amount = int(float(t["Määrä"].replace(",", ".")))
+        timestamp = datetime.strptime(t["Kirjauspäivä"], "%Y/%m/%d").date()
+        archive_id = "TODO: We need the archive ID"
```

Comment:
Change this to something that makes for example sha1 from the information of the transaction. Actual Nordea transaction-ids will not be available.

Reply:
I did as you suggested and went with the SHA1 of the CSV record fields concatenated into a single string.